cmds/cmd_moderation.py

```diff
@@ -164,7 +164,13 @@ class InfractionGenerationError(Exception):
     __slots__ = ()
 
 
-InfractionInfo = Tuple[Optional[discord.Member], InterfacedUser, str, str, Optional[Awaitable[None]]]
+class InfractionInfo(NamedTuple):
+    memeber: Optional[discord.Member]
```

Comment:
memeber?

Reply:
oh god

Reply:
![image](https://user-images.githubusercontent.com/22552258/184753518-5396b00f-d04f-42dd-8814-1514f758b551.png)
i should really add codespell to ci

Reply:
probably a good idea